enemytarget/src/ts/main.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/// <reference path="../tsd/tsd.d.ts" />
+
+import * as React from 'react';
+import * as Reflux from 'reflux';
+import events from 'cu-events';
+import { EnemyTargetStore } from 'cu-stores';
+import { UnitFrame } from 'cu-components';
+
+const enemyTarget: any = EnemyTargetStore.create();
+
+const EnemyTarget = React.createClass({
+
+	// Hook store up to component.  Each time character data is changed,
+	// our state is updated, triggering a render
+	mixins: [
+		Reflux.connect(enemyTarget.store, 'enemyTarget')
+	],
+
+	// Provide an initial state (TODO: is there a better way to do this?)
+	getInitialState: function() {
+		return { enemyTarget: enemyTarget.store.info };
+	},
+
+	componentDidMount() {
+		// Start listening for character events
+		// FIXME: broken, currently no-op
+		enemyTarget.actions.start();
```

Comment:
No, the start logic in the cu-events listener ignores defensive starts.

The correct place for the start is in componentDidMount() the call inside events.on() is to work around the client bug where Calling cuAPI.OnSomething() does not send initial data if you do this outside of the init callback.